server/models/Conversation.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+
+const { ObjectId } = mongoose.Schema.Types;
+const usersArrayBound = (val) => val.length <= 2;
+const willReadArrayBound = (val) => val.length <= 1;
+
+const conversationSchema = new mongoose.Schema(
+  {
+    users: {
+      type: [
+        {
+          type: ObjectId,
+          ref: "User",
+        },
+      ],
+      validate: [usersArrayBound, "Exceeds limit for number of users part of a conversation."],
+      required: true,
+      default: [],
+    },
+    lastMessage: {
+      type: ObjectId,
+      required: false,
+      ref: "Message",
+    },
+    isLastMessageRead: {
```

Comment:
Why didn't we consider adding a 'read' status to each individual message? :)

Reply:
As per our conversation, the read status has been moved to the Messages model. Thanks for the suggestion!